xml/System/AppContext.xml

```diff
@@ -114,7 +114,10 @@
     ```  
   
      Using the `<AppContextSwitchOverrides>` element to define a configuration setting has application scope; that is, it affects only the application.  
-  
+
+    > [!NOTE]    
+    > For information on the switches defined by the .NET Framework, see the [<AppContextSwitchOverrides> element](~/docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md).
```

Comment:
you need to escape the less than operator inside the link text, otherwise it won't render:
![image](https://user-images.githubusercontent.com/12971179/30833871-aa6645a0-a205-11e7-98f8-8168085fec3d.png)